CI Failure

```
[ERROR] org.kitodo.production.forms.createprocess.ProcessDataTabTest.shouldCreateChildProcessTitlePrefixedByParentItile -- Time elapsed: 1.760 s <<< FAILURE!
java.lang.AssertionError
	at org.hibernate.cache.spi.AbstractRegionFactory.verifiedStartStatus(AbstractRegionFactory.java:73)
	at org.hibernate.cache.spi.AbstractRegionFactory.verifyStarted(AbstractRegionFactory.java:62)
	at org.hibernate.cache.spi.support.RegionFactoryTemplate.buildTimestampsRegion(RegionFactoryTemplate.java:66)
	at org.hibernate.cache.internal.EnabledCaching.<init>(EnabledCaching.java:86)
	at org.hibernate.engine.spi.CacheInitiator.initiateService(CacheInitiator.java:30)
	at org.hibernate.engine.spi.CacheInitiator.initiateService(CacheInitiator.java:23)
	at org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.initiateService(SessionFactoryServiceRegistryImpl.java:77)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.service.internal.SessionFactoryServiceRegistryImpl.getService(SessionFactoryServiceRegistryImpl.java:114)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:241)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:450)
	at org.kitodo.data.database.persistence.HibernateUtil.getSessionFactory(HibernateUtil.java:79)
	at org.kitodo.data.database.persistence.HibernateUtil.getSession(HibernateUtil.java:49)
	at org.kitodo.data.database.persistence.BaseDAO.storeObject(BaseDAO.java:518)
	at org.kitodo.data.database.persistence.ProcessDAO.save(ProcessDAO.java:51)
	at org.kitodo.data.database.persistence.ProcessDAO.save(ProcessDAO.java:19)
	at org.kitodo.production.services.data.BaseBeanService.save(BaseBeanService.java:116)
	at org.kitodo.production.services.data.ProcessService.save(ProcessService.java:329)
	at org.kitodo.production.services.data.ProcessService.getProcessDataDirectory(ProcessService.java:847)
	at org.kitodo.production.services.file.FileService.getProcessSubTypeURI(FileService.java:940)
	at org.kitodo.production.services.file.FileService.getMetadataFilePath(FileService.java:736)
	at org.kitodo.production.services.file.FileService.getMetadataFilePath(FileService.java:723)
	at org.kitodo.production.services.data.ProcessService.readMetadataFile(ProcessService.java:988)
	at org.kitodo.production.helper.ProcessHelper.getTitleFromWorkpiece(ProcessHelper.java:550)
	at org.kitodo.production.helper.ProcessHelper.generateAtstslFields(ProcessHelper.java:252)
	at org.kitodo.production.forms.createprocess.ProcessDataTab.generateAtstslFields(ProcessDataTab.java:187)
	at org.kitodo.production.forms.createprocess.ProcessDataTabTest.shouldCreateChildProcessTitlePrefixedByParentItile(ProcessDataTabTest.java:94)

```

is not related to this changes here as this error even happen in main branch inside the slub fork.

--- a/Kitodo/src/main/java/org/kitodo/production/forms/ProjectEditView.java
+++ b/Kitodo/src/main/java/org/kitodo/production/forms/ProjectEditView.java
@@ -156,6 +156,7 @@ public class ProjectEditView extends BaseEditView {
         try {
             this.baseProject = ServiceManager.getProjectService().getById(itemId);
             this.project = ServiceManager.getProjectService().duplicateProject(baseProject);
+            this.workingFolders = new ArrayList<>(project.getFolders());
             this.setSaveDisabled(false);
         } catch (DAOException e) {
             Helper.setErrorMessage(ERROR_DUPLICATE, new Object[] {ObjectType.PROJECT.getTranslationSingular() }, logger,

--- a/Kitodo/src/main/java/org/kitodo/production/services/data/FolderService.java
+++ b/Kitodo/src/main/java/org/kitodo/production/services/data/FolderService.java
@@ -97,6 +97,8 @@ public class FolderService extends BaseBeanService<Folder, FolderDAO> {
         clone.setDpi(source.getDpi().orElse(null));
         clone.setImageSize(source.getImageSize().orElse(null));
         clone.setLinkingMode(source.getLinkingMode());
+        clone.setValidateFolder(source.isValidateFolder());
+        clone.setLtpValidationConfiguration(source.getLtpValidationConfiguration());
         return clone;
     }
 }

--- a/Kitodo/src/main/java/org/kitodo/production/services/data/ProjectService.java
+++ b/Kitodo/src/main/java/org/kitodo/production/services/data/ProjectService.java
@@ -219,12 +219,50 @@ public class ProjectService extends BaseBeanService<Project, ProjectDAO> {
     public Project duplicateProject(Project baseProject) {
         Project duplicatedProject = new Project();
 
+        duplicateGeneralProjectSettings(baseProject, duplicatedProject);
+
+        duplicateFolderSettings(baseProject, duplicatedProject);
+
+        duplicateImportConfiguration(baseProject, duplicatedProject);
+
+        return duplicatedProject;
+    }
+
+    private void duplicateImportConfiguration(Project baseProject, Project duplicatedProject) {
+        duplicatedProject.setDefaultImportConfiguration(baseProject.getDefaultImportConfiguration());
+        duplicatedProject.setDefaultChildProcessImportConfiguration(baseProject.getDefaultChildProcessImportConfiguration());
+    }
+
+    private void duplicateFolderSettings(Project baseProject, Project duplicatedProject) {
+        FolderService folderService = ServiceManager.getFolderService();
+        List<Folder> duplicatedFolders = new ArrayList<>();
+
+        for (Folder folder : baseProject.getFolders()) {
+            Folder duplicatedFolder = folderService.cloneFolder(folder);
+            duplicatedFolder.setProject(duplicatedProject);
+            duplicatedFolders.add(duplicatedFolder);
+        }
+
+        duplicatedProject.setFolders(duplicatedFolders);
+        duplicatedProject.setGeneratorSource(baseProject.getGeneratorSource());
+        duplicatedProject.setMediaView(baseProject.getMediaView());
+        duplicatedProject.setPreview(baseProject.getPreview());
+        duplicatedProject.setAudioMediaView(baseProject.getAudioMediaView());
+        duplicatedProject.setAudioPreview(baseProject.getAudioPreview());
+        duplicatedProject.setAudioMediaViewWaveform(baseProject.isAudioMediaViewWaveform());
+        duplicatedProject.setVideoMediaView(baseProject.getVideoMediaView());
+        duplicatedProject.setVideoPreview(baseProject.getVideoPreview());
+        duplicatedProject.setPreviewHoverMode(baseProject.getPreviewHoverMode());
+    }
+
+    private void duplicateGeneralProjectSettings(Project baseProject, Project duplicatedProject) {
         duplicatedProject.setTitle(baseProject.getTitle() + "_" + Helper.generateRandomString(3));
         duplicatedProject.setClient(baseProject.getClient());
         duplicatedProject.setStartDate(baseProject.getStartDate());
         duplicatedProject.setEndDate(baseProject.getEndDate());
         duplicatedProject.setNumberOfPages(baseProject.getNumberOfPages());
         duplicatedProject.setNumberOfVolumes(baseProject.getNumberOfVolumes());
+        duplicatedProject.setActive(baseProject.isActive());
         duplicatedProject.setDmsImportRootPath(baseProject.getDmsImportRootPath());
         duplicatedProject.setMetsRightsOwner(baseProject.getMetsRightsOwner());
         duplicatedProject.setMetsRightsOwnerLogo(baseProject.getMetsRightsOwnerLogo());
@@ -235,34 +273,6 @@ public class ProjectService extends BaseBeanService<Project, ProjectDAO> {
         duplicatedProject.setMetsPointerPath(baseProject.getMetsPointerPath());
         duplicatedProject.setMetsPurl(baseProject.getMetsPurl());
         duplicatedProject.setMetsContentIDs(baseProject.getMetsContentIDs());
-
-        FolderService folderService = ServiceManager.getFolderService();
-        List<Folder> duplicatedFolders = new ArrayList<>();
-        Folder generatorSource = null;
-        Folder mediaView = null;
-        Folder preview = null;
-
-        for (Folder folder : baseProject.getFolders()) {
-            Folder duplicatedFolder = folderService.cloneFolder(folder);
-            duplicatedFolder.setProject(duplicatedProject);
-            duplicatedFolders.add(duplicatedFolder);
-
-            if (folder.equals(baseProject.getGeneratorSource())) {
-                generatorSource = duplicatedFolder;
-            }
-            if (folder.equals(baseProject.getMediaView())) {
-                mediaView = duplicatedFolder;
-            }
-            if (folder.equals(baseProject.getPreview())) {
-                preview = duplicatedFolder;
-            }
-        }
-        duplicatedProject.setFolders(duplicatedFolders);
-        duplicatedProject.setGeneratorSource(generatorSource);
-        duplicatedProject.setMediaView(mediaView);
-        duplicatedProject.setPreview(preview);
-
-        return duplicatedProject;
     }
 
     /**

--- a/Kitodo/src/test/java/org/kitodo/production/services/data/FolderServiceIT.java
+++ b/Kitodo/src/test/java/org/kitodo/production/services/data/FolderServiceIT.java
@@ -23,6 +23,7 @@ import org.junit.jupiter.api.Test;
 import org.kitodo.MockDatabase;
 import org.kitodo.data.database.beans.Folder;
 import org.kitodo.data.database.beans.Project;
+import org.kitodo.data.database.exceptions.DAOException;
 import org.kitodo.production.services.ServiceManager;
 
 /**
@@ -67,5 +68,28 @@ public class FolderServiceIT {
             assertNotEquals(null, folder.getProject(), "No project assigned");
             assertEquals(project.getTitle(), folder.getProject().getTitle(), "No project assigned");
         }
+    }
+
+    @Test
+    public void testFolderCloning() throws DAOException {
+        FolderService folderService = new FolderService();
+
+        Folder baseFolder = folderService.getById(1);
+        Folder clonedFolder = folderService.cloneFolder(baseFolder);
+
+        assertEquals(baseFolder.getDerivative(), clonedFolder.getDerivative());
+        assertEquals(baseFolder.getDpi(), clonedFolder.getDpi());
+        assertEquals(baseFolder.getFileGroup(), clonedFolder.getFileGroup());
+        assertEquals(baseFolder.getImageSize(), clonedFolder.getImageSize());
+        assertEquals(baseFolder.getLinkingMode(), clonedFolder.getLinkingMode());
+        assertEquals(baseFolder.getLtpValidationConfiguration(), clonedFolder.getLtpValidationConfiguration());
+        assertEquals(baseFolder.getMimeType(), clonedFolder.getMimeType());
+        assertEquals(baseFolder.getPath(), clonedFolder.getPath());
+        assertEquals(baseFolder.getProject(), clonedFolder.getProject());
+        assertEquals(baseFolder.getPath(), clonedFolder.getPath());
+        assertEquals(baseFolder.getUrlStructure(), clonedFolder.getUrlStructure());
+        assertEquals(baseFolder.isCopyFolder(), clonedFolder.isCopyFolder());
+        assertEquals(baseFolder.isCreateFolder(), clonedFolder.isCreateFolder());
+        assertEquals(baseFolder.isValidateFolder(), clonedFolder.isValidateFolder());
     }
 }

--- a/Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java
+++ b/Kitodo/src/test/java/org/kitodo/production/services/data/ProcessServiceIT.java
@@ -24,6 +24,9 @@ import java.io.IOException;
 import java.net.URI;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.sql.Date;
+import java.time.LocalDate;
+import java.time.ZoneId;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashSet;
@@ -47,11 +50,15 @@ import org.kitodo.api.dataformat.Workpiece;
 import org.kitodo.api.dataformat.mets.LinkedMetsResource;
 import org.kitodo.config.ConfigCore;
 import org.kitodo.config.enums.ParameterCore;
+import org.kitodo.data.database.beans.Folder;
 import org.kitodo.data.database.beans.Process;
+import org.kitodo.data.database.beans.Project;
 import org.kitodo.data.database.beans.Role;
 import org.kitodo.data.database.beans.Task;
 import org.kitodo.data.database.beans.User;
 import org.kitodo.data.database.converter.ProcessConverter;
+import org.kitodo.data.database.enums.LinkingMode;
+import org.kitodo.data.database.enums.PreviewHoverMode;
 import org.kitodo.data.database.enums.TaskStatus;
 import org.kitodo.data.database.exceptions.DAOException;
 import org.kitodo.exceptions.FileStructureValidationException;
@@ -854,4 +861,108 @@ public class ProcessServiceIT {
         return false;
     }
 
+    @Test
+    public void testProjectDuplication() throws DAOException {
+        // create new project with all settings set
+        Project baseProject = new Project();
+        baseProject.setTitle("Project for duplication");
+        ClientService clientService = ServiceManager.getClientService();
+        baseProject.setClient(clientService.getById(1));
+        LocalDate localDate = LocalDate.of(2026, 8, 7);
+        baseProject.setStartDate(Date.from(localDate.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant()));
+        baseProject.setEndDate(Date.from(localDate.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant()));
+        baseProject.setNumberOfPages(32);
+        baseProject.setNumberOfVolumes(2);
+        baseProject.setActive(false);
+        baseProject.setDmsImportRootPath("/foo/baar");
+        baseProject.setMetsRightsOwner("Kitodo e.V.");
+        baseProject.setMetsRightsOwnerLogo("kitodo.png");
+        baseProject.setMetsRightsOwnerMail("foo@example.org");
+        baseProject.setMetsRightsOwnerSite("www.kitodo.org");
+        baseProject.setMetsDigiprovPresentation("Foo Baz");
+        baseProject.setMetsDigiprovReference("Baz Foo");
+        baseProject.setMetsPointerPath("pointer path");
+        baseProject.setMetsPurl("purl");
+        baseProject.setMetsContentIDs("contend id");
+
+        // sub folder
+        Folder firstFolder = new Folder();
+        firstFolder.setFileGroup("FOO");
+        firstFolder.setUrlStructure("http://www.example.com/content/$(meta.CatalogIDDigital)/jpgs/max/");
+        firstFolder.setMimeType("image/jpeg");
+        firstFolder.setPath("jpgs/max");
+        firstFolder.setCopyFolder(true);
+        firstFolder.setCreateFolder(true);
+        firstFolder.setDerivative(1.0);
+        firstFolder.setLinkingMode(LinkingMode.EXISTING);
+
+        Folder secondFolder = new Folder();
+        secondFolder.setFileGroup("BAR");
+        secondFolder.setUrlStructure("http://www.example.com/content/$(meta.CatalogIDDigital)/jpgs/default/");
+        secondFolder.setMimeType("image/jpeg");
+        secondFolder.setPath("jpgs/default");
+        secondFolder.setCopyFolder(true);
+        secondFolder.setCreateFolder(true);
+        secondFolder.setDerivative(0.8);
+        secondFolder.setLinkingMode(LinkingMode.ALL);
+
+        baseProject.getFolders().add(firstFolder);
+        baseProject.getFolders().add(secondFolder);
+
+        // set some folder use settings
+        baseProject.setGeneratorSource(secondFolder);
+        baseProject.setMediaView(secondFolder);
+        baseProject.setPreview(firstFolder);
+        baseProject.setPreviewHoverMode(PreviewHoverMode.TOOLTIP_PREVIEW);
+
+        // duplicate the project
+        ProjectService projectService = ServiceManager.getProjectService();
+        Project duplicatedProject = projectService.duplicateProject(baseProject);
+
+        // general settings
+        assertTrue(duplicatedProject.getTitle().startsWith(baseProject.getTitle()));
+        assertEquals(baseProject.getClient(), duplicatedProject.getClient());
+        assertEquals(baseProject.getStartDate(), duplicatedProject.getStartDate());
+        assertEquals(baseProject.getEndDate(), duplicatedProject.getEndDate());
+        assertEquals(baseProject.getNumberOfPages(), duplicatedProject.getNumberOfPages());
+        assertEquals(baseProject.getNumberOfVolumes(), duplicatedProject.getNumberOfVolumes());
+        assertEquals(baseProject.isActive(), duplicatedProject.isActive());
+        assertEquals(baseProject.getDmsImportRootPath(), duplicatedProject.getDmsImportRootPath());
+        assertEquals(baseProject.getMetsRightsOwner(), duplicatedProject.getMetsRightsOwner());
+        assertEquals(baseProject.getMetsRightsOwnerLogo(), duplicatedProject.getMetsRightsOwnerLogo());
+        assertEquals(baseProject.getMetsRightsOwnerMail(), duplicatedProject.getMetsRightsOwnerMail());
+        assertEquals(baseProject.getMetsRightsOwnerSite(), duplicatedProject.getMetsRightsOwnerSite());
+        assertEquals(baseProject.getMetsDigiprovPresentation(), duplicatedProject.getMetsDigiprovPresentation());
+        assertEquals(baseProject.getMetsDigiprovReference(), duplicatedProject.getMetsDigiprovReference());
+        assertEquals(baseProject.getMetsPointerPath(), duplicatedProject.getMetsPointerPath());
+        assertEquals(baseProject.getMetsPurl(), duplicatedProject.getMetsPurl());
+        assertEquals(baseProject.getMetsContentIDs(), duplicatedProject.getMetsContentIDs());
+
+        // simple sub folder check
+        assertEquals(baseProject.getFolders().size(), duplicatedProject.getFolders().size());
+
+        // folder use settings
+        assertEquals(baseProject.getGeneratorSource(), duplicatedProject.getGeneratorSource());
+        assertEquals(baseProject.getMediaView(), duplicatedProject.getMediaView());
+        assertEquals(baseProject.getPreview(), duplicatedProject.getPreview());
+        assertEquals(baseProject.getAudioMediaView(), duplicatedProject.getAudioMediaView());
+        assertEquals(baseProject.getAudioPreview(), duplicatedProject.getAudioPreview());
+        assertEquals(baseProject.isAudioMediaViewWaveform(), duplicatedProject.isAudioMediaViewWaveform());
+        assertEquals(baseProject.getVideoMediaView(), duplicatedProject.getVideoMediaView());
+        assertEquals(baseProject.getVideoPreview(), duplicatedProject.getVideoPreview());
+        assertEquals(baseProject.getPreviewHoverMode(), duplicatedProject.getPreviewHoverMode());
+
+        // project template
+        assertEquals(baseProject.getTemplates(),  duplicatedProject.getTemplates());
+
+        // import configuration
+        assertEquals(
+            baseProject.getDefaultImportConfiguration(),
+            duplicatedProject.getDefaultImportConfiguration()
+        );
+        assertEquals(
+            baseProject.getDefaultChildProcessImportConfiguration(),
+            duplicatedProject.getDefaultChildProcessImportConfiguration()
+        );
+    }
 }

